code/__DEFINES/dcs/signals.dm

```diff
@@ -930,6 +930,9 @@
 /// called on implants, after an implant has been removed: (mob/living/source, silent, special)
 #define COMSIG_IMPLANT_REMOVED "implant_removed"
 
+/// called on implanting a mindshield: 
```

Comment:
Shouldnt you add the args to this comment like previous comsigs?

Reply:
forgot to, thanks